metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/EdtfDatePartNormalizerTest.java

```diff
@@ -42,26 +42,21 @@ public EdtfDatePartNormalizerTest() {
     datePropertyTestCases.put("S.VIII-XV", "07XX/14XX");
```

Comment:
nice reorder.

Reply:
Thanks! This structure will probably change in the future. I'm keeping it for now as a source of truth, since those come from the original handover from Nuno, so that we can easier validate what need to be valid and what not.